.github/workflows/pr.yaml

```diff
@@ -59,16 +59,13 @@ jobs:
       - name: Test Extension APIServer
         timeout-minutes: 2
         run: |
-          POD_NAME=$(kubectl get pod -o name -l app.kubernetes.io/name=konk-service )
-          until cat test/apiserver/sample/contact.yaml | kubectl exec -i $POD_NAME -- kubectl apply -f -
-          do
-            sleep 1
-          done
-          until kubectl exec -it $POD_NAME -- kubectl get contact contact-example
-          do
-            sleep 1
-          done
-          kubectl exec -it $POD_NAME -- kubectl get crd appconfigs.stable.example.com websites.extensions.example.com
+          make test-example-apiserver
+      - name: Push images
+        timeout-minutes: 3
+        run: |
+          echo TODO add push credentials to repo
+          # make docker-push
+          # make -C test/apiserver push
```

Comment:
I would put this in a different workflow that only runs on `main`.

Reply:
done